### PR TITLE
Make revert messages consistent for DAO related contracts

### PR DESCRIPTION
**Pull request by manhlx3006**

### CODE REVIEW

File: contracts/sol6/Dao/KyberDAO.sol [hunk 1]
```diff
@@ -490,7 +490,7 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5 {
     }
 
     // return list campaign ids for epoch, excluding non-existed ones
-    function getListCampIDs(uint256 epoch) external view returns (uint256[] memory campaignIDs) {
+    function getListCampaignIDs(uint256 epoch) external view returns (uint256[] memory campaignIDs) {
```
Comment (ilanDoron) [inline contracts/sol6/Dao/KyberDAO.sol:493]: is this API supported on Jarvis?

should consult Victor for changing this

I support this change!

File: contracts/sol6/Dao/KyberDAO.sol [hunk 2]
```diff
@@ -770,7 +770,7 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5 {
                 // so network fee should be less than 50%
                 require(
                     options[i] < BPS / 2,
-                    "validateParams: Fee campaign option value is too high"
+                    "validateParams: fee campaign option value is high"
```
Comment (ilanDoron) [inline contracts/sol6/Dao/KyberDAO.sol:773]: lets make this clearer.


validateParams: network fee must be smaller then BPS / 2

File: contracts/sol6/Dao/KyberDAO.sol [hunk 3]
```diff
@@ -781,7 +781,7 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5 {
                     getRebateAndRewardFromData(options[i]);
                 require(
                     rewardInBps.add(rebateInBps) <= BPS,
-                    "validateParams: RR values are too high"
+                    "validateParams: RR values are high"
```
Comment (ilanDoron) [inline contracts/sol6/Dao/KyberDAO.sol:784]: validateParams: rebate + reward must be smaller then BPS.

File: contracts/sol6/Dao/KyberDAO.sol (hunk 1 above)
Comment (manhlx3006) [reply contracts/sol6/Dao/KyberDAO.sol:493]: Victor has not updated Jarvis with newest APIs for submit campaign, get campaign details. So he will update Jarvis once we finalize

Comment (ilanDoron) [reply contracts/sol6/Dao/KyberDAO.sol:493]: OK
Great

**Pull request merged**